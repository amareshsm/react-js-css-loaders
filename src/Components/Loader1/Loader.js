import React from 'react'
import styles from './style.css'
const Loader1 = (props) => (
  <>
    <style>{`${styles}`}</style>
    <div className="loader-div one">
      <div className="loader1"></div>
    </div>
  </>
)

export default Loader1
