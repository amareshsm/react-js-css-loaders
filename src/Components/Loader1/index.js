export { default as Moon } from './Loader'
