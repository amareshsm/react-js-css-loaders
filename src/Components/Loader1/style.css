* {
    padding: 0px;
    margin: 0px;
    box-sizing: border-box;
}

.loader-div {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 300px;
    width: 380px;
    margin: 8px;
    border-radius: 4px;
}
.loader-div.one {
    background-color: lightyellow;
}

.loader1 {
    width: 90px;
    height: 90px;
    border-top: 4px solid blue;
    border-radius: 100%;
    animation: spin 2s linear infinite;
}

@keyframes spin {
    0% {
        transform: rotate(0deg);
    }
    100% {
        transform: rotate(360deg);
    }
}
