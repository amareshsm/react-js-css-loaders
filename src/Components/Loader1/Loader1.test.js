import React from 'react'
import renderer from 'react-test-renderer'
import { Loader1 } from '../Loader1'
it('renders correctly', () => {
  const tree = renderer.create(<Loader1 />).toJSON()
  expect(tree).toMatchSnapshot()
})
