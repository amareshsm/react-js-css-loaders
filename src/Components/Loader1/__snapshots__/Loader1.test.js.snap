// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="loader-div one"
  >
    <div
      className="loader1"
    />
  </div>,
]
`;
